src/routes/transactions/entities/multisig-transaction.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class ExecutionInfo {
+  @ApiProperty()
+  type: string;
+  @ApiProperty()
+  nonce: number;
+  @ApiProperty()
+  confirmationsRequired: number;
+  @ApiProperty()
+  confirmationsSubmitted: number;
+  @ApiPropertyOptional()
+  missingSigners?: string[];
+}
+
+export class TransactionInfo {
+  @ApiProperty()
+  type: string;
+}
+
+export class TransactionSummary {
+  @ApiProperty()
+  id: string;
+  @ApiPropertyOptional()
+  timestamp?: number;
+  @ApiProperty()
+  txStatus: string;
+  @ApiProperty()
+  txInfo: TransactionInfo;
+  @ApiProperty()
+  executionInfo: ExecutionInfo;
```

Comment:
I think that this is optional and there is other one that is `SafeAppInfo`.

Reply:
True, I forgot to include it 😟 maybe because `SafeAppInfo` wasn't included in any of the transactions I used to test the functionality. But I think this deserves special attention because it seems there's some retrieval logic (see [safe_app_info.rs](https://github.com/safe-global/safe-client-gateway/blob/main/src/routes/transactions/converters/safe_app_info.rs)) and I'm not sure we need to change something in the domain because `origin` seems to be a JSON field. 

I included the entities in [ea6fec6](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/ea6fec62a715b307096688b11b1a4da1b39de46f) and I think it's better to let this as a TODO and tackle it in a separate PR, wdyt? (Any help from @fmrsabino is really appreciated since he probably has more context on this)

Reply:
I would like to include it in this PR because is part of the `multisigtransaction` endpoint, if we forget include it we could break things on the client side. 
I was searching an example but while the tx-service returns the `origin`, the client-gateway is not including the `safe_app_info`.

Reply:
Ok, I was suggesting it to avoid doing too many things in this PR, it is complex enough I think. Closing this PR doesn't mean this feature/route is ready for production if we create an additional issue to tackle it. But I'm fine doing it here if you prefer 🙂 I'd like to have some inputs from @fmrsabino regarding this if possible before implementing the `origin` mapping.

Reply:
Done 🙂 https://github.com/5afe/safe-client-gateway-nest/issues/201